docs/concepts/artifacts.md

```diff
@@ -0,0 +1,228 @@
+    
+    Unlike the `print()` command, where you can concatenate multiple calls to include additional items in a report, within a task, these commands must be used multiple times if necessary. 
+    
+    To create artifacts like reports or summaries using `create_markdown_artifact()`, compile your message string separately and then pass it to `create_markdown_artifact()` to create the complete artifact.
+
+### Creating Link Artifacts
+
+To create a link artifact, use the `create_link_artifact()` function.
+
+```python
+from prefect import flow, task
+from prefect.experimental.artifacts import create_link_artifact
+import pendulum
+
+@task
+def my_first_task():
+    now = pendulum.now('UTC').to_datetime_string()
+    data_variability = 1764 ** 0.5
+    if data_variability >= 42:
+        create_link_artifact(
+            key="irregular-data",
+            link=f"https://nyc3.digitaloceanspaces.com/my-bucket-name/highly_variable_data_{now}.csv",
+            description="## Highly variable data",
+        )
+
+@task
+def my_second_task():
+    now = pendulum.now('UTC').to_datetime_string()
+    prediction_accuracy = 0.42
+    if prediction_accuracy >= 0.50:
+        create_link_artifact(
+            key="irregular-data",
+            link=f"https://nyc3.digitaloceanspaces.com/my-bucket-name/low_pred_data_{now}.csv",
+            description="# Low prediction accuracy",
+        )
+
+@flow
+def my_flow():
+    my_first_task()
+    my_second_task()
+
+if __name__ == "__main__":
+    my_flow()
+```
+
+!!! tip Specify multiple artifacts with the same key for artifact lineage
+    You can specify multiple artifacts with the same key to more easily track something very specific that you care about, such as irregularities in your data pipeline. 
+
+After running the above flows, you can find your new artifacts in the Artifacts page of the UI. You can click into the "irregular data" artifact and see all versions of it, along with custom descriptions and links to the relevant data.
+
+Here, you'll also be able to view information about your artifact such as its associated flow run or task run id, previous and future versions of the artifact (multiple artifacts can have the same key in order to show lineage), the data you've stored (in this case a markdown-rendered link), an optional markdown description, and when the artifact was created or updated.
+
+To make the links more readable for you and your collaborators, you can pass in a `link_text` argument for your link artifacts:
+
+```python
+from prefect import flow
+from prefect.experimental.artifacts import create_link_artifact
+
+@flow
+def my_flow():
+    create_link_artifact(
+        key="my-important-link",
+        link="https://www.prefect.io/",
+        link_text="Prefect",
```

Comment:
```suggestion
        link_text="Prefect",
        description="Prefect link"
```

Reply:
You don't need to specify a description, so I'm trying to show a variety of examples.

Reply:
That makes sense.  I updated the explanation suggestion above to reflect that.

Reply:
Nice 🧠